examples/posenet/cameraGui.js

```diff
@@ -0,0 +1,53 @@
+let guiState;
+function setupGui(){
+  guiState = {
+    algorithm: 'multi-pose',
+    model: 1.01,
+    outputStride: 16,
+    scaleFactor: 0.5,
+    scoreThreshold: 0.5,
+    multiPoseDetection: {
+      maxDetections: 5,
+      nmsRadius: 30.0,
+    },
+  };
+
+  const gui = new dat.GUI({width: 300});
+  gui.domElement.id = 'gui';
+  const algorithm = gui.add(guiState, 'algorithm', ['single-pose', 'multi-pose']).onChange((algorithm)=>{
+    guiState.algorithm = algorithm;
+    main();
```

Comment:
It looks like we need to re-create all things when every single parameter change. Is it necessary for all cases?